content/id/docs/tasks/manage-daemon/rollback-daemon-set.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Melakukan Rollback pada DaemonSet
+content_type: task
+weight: 20
+min-kubernetes-server-version: 1.7
+---
+
+<!-- overview -->
+
+Laman ini memperlihatkan bagaimana caranya untuk melakukan _rollback_ pada sebuah {{< glossary_tooltip term_id="daemonset" >}}.
+
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+Sebelum lanjut, alangkah baiknya jika kamu telah mengetahui cara
+untuk [melakukan _rolling update_ pada sebuah DaemonSet](/docs/tasks/manage-daemon/update-daemon-set/).
+
+<!-- steps -->
+
+## Melakukan _rollback_ pada DaemonSet
+
+### Langkah 1: Dapatkan nomor revisi DaemonSet yang ingin dikembalikan
+
+Lompati langkah ini jika kamu hanya ingin kembali (_rollback_) ke revisi terakhir.
+
+Perintah di bawah ini akan memperlihatkan daftar semua revisi dari DaemonSet:
+
+```shell
+kubectl rollout history daemonset <nama-daemonset>
+```
+
+Perintah tersebut akan menampilkan daftar revisi seperti di bawah:
+
+```
+daemonsets "<nama-daemonset>"
+REVISION        CHANGE-CAUSE
+1               ...
+2               ...
+...
+```
+
+* Alasan perubahan (_change cause_) kolom di atas merupakan salinan dari anotasi `kubernetes.io/change-cause` yang berkaitan dengan revisi pada DaemonSet. Kamu boleh menyetel _flag_ `--record=true` melalui `kubectl` untuk merekam perintah yang dijalankan akibat dari anotasi alasan perubahan.
```

Comment:
I just learn official word https://kbbi.kemdikbud.go.id/entri/menyetel

Reply:
oh I was lucky, didn't check properly on KBBI for this 😓